plugin/__tests__/.prettierrc

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
this one is important, it's for babel plugin tests

Reply:
Sounds good, I reverted it. Do you mind telling me how you are running the command? Something like `npx prettier * --write --config .prettierrc` from the `plugin/__tests__`

Reply:
You don't run it, it's for predictable output for `babel-plugin-tester`. It runs automatically with `yarn test`

Reply:
https://github.com/babel-utils/babel-plugin-tester?tab=readme-ov-file#formatting-output-with-prettier

Reply:
I see, thanks for the input!